staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -546,6 +546,11 @@ const (
 	JobReasonBackoffLimitExceeded string = "BackoffLimitExceeded"
 	// JobReasponDeadlineExceeded means job duration is past ActiveDeadline
 	JobReasonDeadlineExceeded string = "DeadlineExceeded"
+	// JobReasonMaxFailedIndexesExceeded indicates that an indexed of a job failed
+	// https://kep.k8s.io/3850
+	JobReasonMaxFailedIndexesExceeded string = "MaxFailedIndexesExceeded"
+	// JobReasonFailedIndexes means Job has failed indexes.
+	JobReasonFailedIndexes string = "FailedIndexes"
```

Comment:
Indicate that they are beta

Reply:
I missed that once I noticed lgtm :) Opened small cleanup: https://github.com/kubernetes/kubernetes/pull/121471.